core/src/smartcontracts/wasm.rs

```diff
@@ -1004,7 +1004,7 @@ where
 
 /// Marker trait to auto-implement [`import_traits::SetPermissionTokenSchema`] for a concrete [`Runtime`].
 ///
-/// Useful because in *Validator* exist more entrypoints than just `migrate()` which is the
+/// Useful because in *Executor* exist more entrypoints than just `migrate()` which is the
```

Comment:
This sounds more correct to me:
```suggestion
/// Useful because *Executor* exposes more entrypoints than just `migrate()` which is the
```